<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMusic</class>
 <widget class="QWidget" name="MyMusic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>800</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="play_page">
    <widget class="QLabel" name="perfect_lable">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>460</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>160</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="main_back">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>560</width>
       <height>800</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="bar_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>484</y>
       <width>560</width>
       <height>190</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="left_gif">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>429</y>
       <width>140</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="up_gif">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>429</y>
       <width>140</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="down_gif">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>429</y>
       <width>140</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="right_gif">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>429</y>
       <width>140</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="down_cover">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>575</y>
       <width>560</width>
       <height>245</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="up_dis">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>576</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="left_dis">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>576</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="down_dis">
     <property name="geometry">
      <rect>
       <x>288</x>
       <y>576</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="right_dis">
     <property name="geometry">
      <rect>
       <x>424</x>
       <y>576</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_button">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>630</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pattern_label">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>0</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="combo_left">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>310</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="combo_right">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>310</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="cheng_label">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>310</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="check_label_right">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>637</y>
       <width>140</width>
       <height>180</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="check_label_left">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>637</y>
       <width>140</width>
       <height>180</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="check_label_up">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>637</y>
       <width>140</width>
       <height>180</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="check_label_down">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>637</y>
       <width>140</width>
       <height>180</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="HP">
     <property name="geometry">
      <rect>
       <x>603</x>
       <y>472</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QProgressBar {
   border: 2px solid grey;
   border-radius: 5px;
   background-color: white;
}
 
QProgressBar::chunk {
   background-color: red;
   width: 20px;
}
 
QProgressBar {
   border: 2px solid grey;
   border-radius: 5px;
   text-align: center;
}</string>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="right_right_label">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>0</y>
       <width>290</width>
       <height>800</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="menlian_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>560</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="end_page">
    <widget class="QLabel" name="end_label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>271</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;结束&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="end_close">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>560</y>
       <width>250</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭游戏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="to_main">
     <property name="geometry">
      <rect>
       <x>302</x>
       <y>457</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>回到主窗口</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="qiege_page">
    <widget class="QLabel" name="background">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>4</y>
       <width>851</width>
       <height>801</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="dianbo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>315</y>
       <width>841</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
